src/renderers/WebGLRenderTarget.js

```diff
@@ -12,25 +12,16 @@ THREE.WebGLRenderTarget = function ( width, height, options ) {
 
 	options = options || {};
 
-	this.wrapS = options.wrapS !== undefined ? options.wrapS : THREE.ClampToEdgeWrapping;
-	this.wrapT = options.wrapT !== undefined ? options.wrapT : THREE.ClampToEdgeWrapping;
-
-	this.magFilter = options.magFilter !== undefined ? options.magFilter : THREE.LinearFilter;
-	this.minFilter = options.minFilter !== undefined ? options.minFilter : THREE.LinearMipMapLinearFilter;
-
-	this.anisotropy = options.anisotropy !== undefined ? options.anisotropy : 1;
-
-	this.offset = new THREE.Vector2( 0, 0 );
-	this.repeat = new THREE.Vector2( 1, 1 );
-
-	this.format = options.format !== undefined ? options.format : THREE.RGBAFormat;
-	this.type = options.type !== undefined ? options.type : THREE.UnsignedByteType;
+	this.texture = new THREE.Texture(undefined, undefined, 
+																	 options.wrapS, options.wrapT,
+																	 options.magFilter, 
+																	 options.minFilter !== undefined ? options.minFilter : THREE.LinearFilter,
```

Comment:
You have some formatting issues. There are ~17 leading tabs here, and whitespace issues elsewhere.

If you would like some help with formatting, see the Wiki article [Mr.doob's-Code-Style™](https://github.com/mrdoob/three.js/wiki/Mr.doob%27s-Code-Style™).